lib/customer.rb

```diff
@@ -0,0 +1,34 @@
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
Just a note, you don't need to make email and address accessors, they could simply be readers.  You want to be a little cautious about giving the user more access to the internals of your class than they need